MSAL/src/public/MSALAccount.h

```diff
@@ -28,6 +28,15 @@
 #import <Foundation/Foundation.h>
 
 @class MSALAccountId;
+@class MSALTenantProfile;
+
+typedef NS_ENUM(NSInteger, MSALAccountType)
```

Comment:
What is the difference between MSALAccount & MSIDAccountType?

Reply:
removed the MSALAccountType in my latest change :)